src/main/java/Deadline.java

```diff
@@ -0,0 +1,42 @@
+import java.io.PrintWriter;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task {
+    protected LocalDateTime by;
```

Comment:
Similar to the Event class, I don't think that the name "by" is very comprehensive unless I know how to create a Deadline object in the CLI.